Tasks/NuGetToolInstallerV1/task.loc.json

```diff
@@ -15,7 +15,7 @@
   "version": {
     "Major": 1,
     "Minor": 149,
```

Comment:
Unless you are planning to cherry pick, shouldn't the minor version be 150?

Reply:
nvm, I think you already resolved this in later commits

Reply:
I updated all the task versions at once in a different commit - the task.loc files were updated here because they sync with task.json on build, basically I guess they weren't synced before they were checked in.